src/api.js

```diff
@@ -9,3 +9,13 @@ export const getConferenceList = async () => {
     throw error;
   }
 };
+
+export const getConferenceById = async (id) => {
+  try {
+    const response = await axios.get(`http://localhost:8080/conferences/${id}`);
```

Comment:
Maybe something for future PRs: All these URLs probably should be put in to a config file where they can be changed and maintained more easily :) but not really part of this PR :)

Reply:
ok we'll look into doing this in the future, thank you!!